### PR TITLE
specify platform in all dockerfiles

### PR DESCRIPTION
**Pull request by emileten**
This came back to bite me so I specified the platform in all dockerfiles. 

https://github.com/NASA-IMPACT/active-maap-sprint/issues/443#issuecomment-1443821296

https://github.com/MAAP-Project/veda-data-pipelines/pull/6

### CODE REVIEW

**Comment by emileten** [comment]
@jjfrench got a second to look at this ? Should be pretty quick. Sorry or all the PRs, it's probably hard to follow. I merged this one to https://github.com/MAAP-Project/MAAP-data-pipelines/pull/1.

**Review by jjfrench — APPROVED** [review]
